sway/commands.c

```diff
@@ -145,14 +184,19 @@ static enum cmd_status cmd_bindsym(int argc, char **argv) {
 }
 
 static enum cmd_status cmd_exec_always(int argc, char **argv) {
+	DEFER();
 	if (!checkarg(argc, "exec_always", EXPECTED_MORE_THAN, 0)) {
 		return CMD_FAILURE;
 	}
-	if (!config->active) {
-		return CMD_DEFER;
-	}
+	// Put argument into cmd array
+	char *tmp = join_args(argv, argc);
+	char cmd[4096];
+	strcpy(cmd, tmp);
+	free(tmp);
+	
+	char *args[] = {"sh", "-c", cmd, 0 };
 
-	pid_t pid = fork();
+	pid_t pid = vfork();
```

Comment:
well i guess rearranging the sway_log before the fork would also fix it too,
though since all we do is fork/exec thats what vfork was made to do so either way is fine.